### ISSUE TITLE
chaning to relative path does not work and prevents convertion to biblatex

### ISSUE THREAD

**Issue by JoKalliauer** [JabRef/jabref#6223]
```
JabRef 5.1--2020-03-23--67daf36
Windows 10 10.0 amd64 
Java 14
```
```tex
% Encoding: UTF-8

@Article{kauert2011direct,
  author    = {Kauert, Dominik J. and Kurth, Thomas and Liedl, Tim and Seidel, Ralf},
  journal   = {Nano Letters},
  title     = {Direct mechanical measurements reveal the material properties of three-dimensional {DNA} origami},
  year      = {2011},
  issn      = {1530-6984},
  month     = dec,
  number    = {12},
  pages     = {5558--5563},
  volume    = {11},
  adsnote   = {Provided by the SAO/NASA Astrophysics Data System},
  adsurl    = {http://adsabs.harvard.edu/abs/2011NanoL..11.5558K},
  doi       = {10.1021/nl203503s},
  file      = {:../../Literature/DNA/Kauert_Kurth_Liedl_Seidel_2011_Nano letters_Main.pdf:PDF;:../../Literature/DNA/Kauert_Kurth_Liedl_Seidel_2011_Nano letters_SupportingInfo.pdf:PDF;\:..\\\\..\\\\Literature\\\\DNA\\\\Kauert_Kurth_Liedl_Seidel_2011_Nano letters_Main.pdf\:PDF\;\:..\\\\..\\\\Literature\\\\DNA\\\\Kauert_Kurth_Liedl_Seidel_2011_Nano letters_SupportingInfo.pdf\:PDF:\:../../Literature/DNA/Kauert_Kurth_Liedl_Seidel_2011_Nano letters_Main.pdf\:PDF\;\:../../Literature/DNA/Kauert_Kurth_Liedl_Seidel_2011_Nano letters_SupportingInfo.pdf\:PDF:PDF},
  groups    = {DNA, YoungsModulus, DNAOrigami},
  keywords  = {rank1},
  publisher = {ACS Publications},
  ranking   = {rank1},
}

@Comment{jabref-meta: databaseType:bibtex;}
```

1. open [bibtex2biblatexBug.bib.txt](https://github.com/JabRef/jabref/files/4410484/bibtex2biblatexBug.bib.txt)
2. clean up using following settings 
![Screenshot 2020-03-31 18 57 27](https://user-images.githubusercontent.com/15687581/78054678-b8b47b80-7382-11ea-8527-77c5e7045f28.png)

nothing happens (also journal should be converted to journaltitle) and if you mark several files no file (after that file) get changed. (No warning no errormessage)

If I disable "change to relative path" it works again.

The strange thing you newer know it just needs some time or it does not work.

**Comment by Siedlerchr** [comment]
We recently discovered and fixed a bug related to cleanup. Can you please try again with the latest master
 https://builds.jabref.org/master and report back if it works for you.

**Comment by JoKalliauer** [comment]
```
JabRef 5.2--2020-09-04--3db4313
Windows 10 10.0 amd64 
Java 14.0.2
```

With the settings described above `journal` still dont get converted to `journaltitle`.

![2020-09-05](https://user-images.githubusercontent.com/15687581/92312302-bd2f3100-efbf-11ea-8297-90ae65aba01f.png)

**Comment by JoKalliauer** [comment]
seems to be fixed in `JabRef 5.2--2020-12-24--6a2a512`
